packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -348,7 +348,17 @@ class AnimationController extends Animation<double>
   /// after this method is called.
   @override
   void dispose() {
+    assert(() {
+      if (_ticker == null) {
+        throw new FlutterError(
+          'AnimationController.dispose() called more than once.\n'
+          'A given AnimationController for $debugLabel cannot be disposed more than once.'
```

Comment:
I think the "for $debugLabel" part here is confusing the way you have it phrased. It sounds like you're saying that others can be. Maybe remove that part of the sentence and add a second sentence saying "However, the AnimationController for $debugLabel was." or something.